src/Makefile

```
@@ -325,10 +325,19 @@ ifndef V
         @printf '    %b %b\n' $(LINKCOLOR)INSTALL$(ENDCOLOR) $(BINCOLOR)$(1)$(ENDCOLOR) 1>&2
         @$(INSTALL) $(1) $(2)
     endef
+
+    define CREATE_SYMLINK
+        @ln -sf $(1) $(2)/$(3)
+        @printf '    %b %b %b %b %b %b %b\n' $(LINKCOLOR)CREATE SYMLINK$(ENDCOLOR) from $(BINCOLOR)$(3)$(ENDCOLOR) to $(BINCOLOR)$(1)$(ENDCOLOR)  1>&2
```

Suggesting:

      INSTALL valkey-cli
      ...
      INSTALL SYMLINK redis-cli -> valkey-cli

It looks slightly better IMO. :) The arrow is similar to how they are displayed by `ls -l`.

```suggestion
        @printf '    %b %b %b %b %b %b %b\n' $(LINKCOLOR)INSTALL SYMLINK$(ENDCOLOR) $(BINCOLOR)$(3)$(ENDCOLOR) -> $(BINCOLOR)$(1)$(ENDCOLOR)  1>&2
```

---

Yeah, it looks much better 👍